internal/resource/resource.go

```diff
@@ -301,6 +302,9 @@ func NewResource(ctx context.Context, slice *apiv1.ResourceSlice, index int) (*R
 	const disableUpdatesKey = "eno.azure.io/disable-updates"
 	res.DisableUpdates = anno[disableUpdatesKey] == "true"
 
+	const forceKey = "eno.azure.io/force"
```

Comment:
any reason to use "force" to represent disable merge behavior? can we use "disable-merge" like disable-updates above? or force-recreate to make it self-explanatory

Reply:
That's a great question, I'm not 100% sold on "force" - I picked it mostly because it's already a concept in helm.

That said, I suspect we may have other functionality that makes sense to include under "force" in the future, rather than having separate annotations. Support for immutable configmaps comes to mind, or playing nice with `generateName` more broadly.

Reply:
I think force is fine to use for replacing. force-recreate might be a bit more self-explanatory, but I agree it makes sense to keep just one annotation if we want to avoid having multiple similar ones.